src/Operator/Constructors.jl

```diff
@@ -780,6 +781,186 @@ function advectionoperator(basis::AbstractBasis, mesh::Mesh, wind = [1, 1])
     advection = Operator(advectionweakform, mesh, inputs, outputs)
     return advection
 end
+
+"""
+```julia
+supgadvectionoperator(basis, mesh, wind)
+```
+Convenience constructor for SUPG advection operator
+
+# Weak form: Right hand side
+- ``\\int wind \\nabla v u - wind wind τ \\nabla v \\nabla u``
+
+# Arguments:
+- `basis`: basis for all operator fields to use
+- `mesh`:  mesh for operator
+- `wind`:  advection speed in 2D
+- `τ`:     scaling for SUPG
+
+# Returns:
+- SUPG advection operator with basis on mesh
+
+# Example:
+```jldoctest
+# supg advection operator
+mesh = Mesh2D(1.0, 1.0);
+mapping = nothing
+#mapping = sausage_transformation(9);
+basis = TensorH1LagrangeBasis(3, 4, 1, mesh.dimension, collocatedquadrature = false, mapping = mapping)
+wind = [1, 1]
+supgadvection = LFAToolkit.supgadvectionoperator(basis, mesh, wind);
+
+# verify
+println(supgadvection)
+
+# output
+
+finite element operator:
+2d mesh:
+    dx: 1.0
+    dy: 1.0
+
+2 inputs:
+operator field:
+  tensor product basis:
+    numbernodes1d: 3
+    numberquadraturepoints1d: 4
+    numbercomponents: 1
+    dimension: 2
+  evaluation mode:
+    modes:
```

Comment:
This is your doctest bug.
```suggestion
  evaluation modes:
```